wfe-core/pom.xml

```diff
@@ -25,6 +25,11 @@
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-context</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
This dependency pulls out spring-web fully.
This seems excessively only for browser detection. 
May be something like https://stackoverflow.com/a/3540295 will be sufficient?